File: tests/CtorNoName.pas
namespace Demo;

type
  CtorNoName = public class
  public
    constructor;
    constructor(flag: Boolean);
    destructor;
  end;

implementation

constructor CtorNoName.Create;
begin
  inherited;
end;

constructor CtorNoName.Create(flag: Boolean);
begin
  inherited;
end;

destructor CtorNoName.Destroy;
begin
  inherited;
end;

File: tests/CtorNoName.cs
namespace Demo {
    public partial class CtorNoName {
        public void Create();
        public void Create(bool flag);
        public void Destroy();
        public void Create() {
            base.Create();
        }
        public void Create(bool flag) {
            base.Create(flag);
        }
        public void Destroy() {
            base.Destroy();
        }
    }
}
